assets/scss/components/_activity.scss

```diff
@@ -0,0 +1,33 @@
+.activity {
+	padding: 100px 20px;
+
+	h3 {
```

Comment:
I probably would have given this a class to make it more flexible.  What if someone changes this to an h2 or h4 inside that activity class?